types/markdown-it-emoji/tsconfig.json

```diff
@@ -14,7 +12,23 @@
     },
     "files": [
         "index.d.ts",
-        "test/markdown-it-emoji-cjs.ts",
+        "index.d.mts",
+        "dist/bare.cjs.d.ts",
+        "dist/full.cjs.d.ts",
+        "dist/index.cjs.d.ts",
+        "dist/light.cjs.d.ts",
+        "dist/markdown-it-emoji-bare.d.ts",
+        "dist/markdown-it-emoji-bare.min.d.ts",
+        "dist/markdown-it-emoji-light.d.ts",
+        "dist/markdown-it-emoji-light.min.d.ts",
+        "dist/markdown-it-emoji.d.ts",
+        "dist/markdown-it-emoji.min.d.ts",
+        "lib/bare.d.mts",
+        "lib/full.d.mts",
+        "lib/light.d.mts",
+        "test/markdown-it-emoji.ts",
+        "test/markdown-it-emoji.mts",
         "test/markdown-it-emoji-global.ts"
```

Comment:
not sure if this is expected, but currently it needs to list `"test/markdown-it-emoji-global.ts"` in `files` of main tsconfig too. ESLint doesn't seem to pick up `files` from tsconfig files in `references`

Reply:
seems restriction because of ts-eslint (#68908)